src/tests/test_asyncio.py

```diff
@@ -376,12 +384,15 @@ def test_await_pyproxy_eval_async(selenium):
             """
             let finally_occurred = false;
             let err_occurred = false;
-            let c = pyodide._module.pyodide_py._base.eval_code_async("raise ValueError('hi')");
+            let eval_code_async = pyodide.pyodide_py.eval_code_async;
```

Comment:
It's not actually important for user code, it increases the reference count of `eval_code_async` but that guy is never going to be freed, so I don't think it matters for practical purposes if there are tons of leaked references to it. We have to do it in the tests because I am specifically measuring for leaked PyProxies.

On the other hand, there will be cases where people use methods of objects that they actually want to be reclaimed and in that case this is very annoying.

I have a complete implementation of #1617 too, oddly enough it is blocked on this PR.